Assignment2/jmdu99-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,52 @@
+{
+	"@context": {
+	"rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "prefix": "http://example.org/assignment2b/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "name": "rdfs:label",
+    "hometown": "rdfs:label",
+    "age": "rdfs:label",
+    "university":"rdfs:label",
+    "parents":"rdfs:label",
+    "mother":"rdfs:label",
+    "father":"rdfs:label",
+    "subjects":"rdfs:label",
+    "subject":"rdfs:label",
+    "teachers":"rdfs:label",
+    "teacher":"rdfs:label",
+    "description":"rdfs:comment"
+  },
+  "@id": "prefix:jmdu99",
+  "@type": "prefix:",
+  "name": {"label":"Nombre", "@value":"José Manuel Díaz Urraco"},
+	"hometown":{"@value":"Boadilla del Monte"},
+    "age":{"@value":21,"@type":"xsd:integer"},
+    "university": {"@value":"UPM"},
+  "parents": {
+		"mother": {"@value":"Marina Urraco Cendrero"},
+		"father": {"@value":"José Manuel Díaz Soriano"}
+	},
+	"subjects": {
+		"subject": [{
+				"name": {"@value":"Gestión de Procesos de Tecnologías de la Información"},
+				"teachers": {
+					"teacher": [
```

Comment:
You don't need it, just leave the [
Be careful with the end of the array.
Do the same with the next "teacher" attribute

Reply:
I just leave the [ but the JSON-LD playground shows me an error :(

Reply:
I meant, delete -> { "teacher": 
and delete the corresponding closing "}"
Try it ;-)

Reply:
Ah okey. Done it.